### ISSUE TITLE
MacOS build failed

### ISSUE THREAD

**Issue by mdsohelmia** [master-of-zen/Av1an#879]
= note: ld: library 'vapoursynth' not found
          clang: error: linker command failed with exit code 1 (use -v to see invocation)

The following warnings were emitted during compilation:

warning: av1an@0.4.3: could not find repository at '/Users/sohelmia/.cargo/registry/src/index.crates.io-6f17d22bba15001f/av1an-0.4.3'; class=Repository (6); code=NotFound (-3)
warning: av1an@0.4.3: VERGEN_GIT_COMMIT_DATE set to default
warning: av1an@0.4.3: VERGEN_GIT_SHA set to default

error: could not compile `av1an` (bin "av1an") due to 1 previous error
error: failed to compile `av1an v0.4.3`, intermediate artifacts can be found at `/var/folders/2k/f3rpm1rn14sdn_ysz8gjkkc40000gn/T/cargo-installlkD4eZ`.
To reuse those artifacts with a future compilation, set the environment variable `CARGO_TARGET_DIR` to that path.

**Comment by FreezyLemon** [comment]
Probably a duplicate of #815 (assuming you used homebrew). Probably fixed soon, just waiting on the vapoursynth crate to release the update.
